src/views/tutorials-landing/tutorials-landing.module.css

```diff
@@ -1,48 +1,106 @@
-.root {
-	--padding-left-right: 56px;
-	--text-max-width: 1240px;
+.variables {
```

Comment:
Not positive where to attribute this, but there are a few breakpoints where the cards are awkwardly tall:

<img width="531" alt="CleanShot 2023-06-05 at 18 01 40@2x" src="https://github.com/hashicorp/dev-portal/assets/2105067/68b2df9a-e6e9-42e6-b3ba-144a5ee4a813">
<img width="534" alt="CleanShot 2023-06-05 at 18 01 12@2x" src="https://github.com/hashicorp/dev-portal/assets/2105067/2993b05e-0008-471a-84f7-c926b97305cd">
<img width="986" alt="CleanShot 2023-06-05 at 17 58 29@2x" src="https://github.com/hashicorp/dev-portal/assets/2105067/dc8aaa56-e779-4335-ac34-e027b509fb6f">

Reply:
To note -- these things will be refined in vqa/polish PRs!